demo/src/main/java/com/example/demo/controller/ApiController.java

```diff
@@ -1,35 +1,33 @@
-package com.example.demo.Controller;
-
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.bind.annotation.GetMapping;
+package com.example.demo.controller;
 
+import com.example.demo.models.User;
 import com.example.demo.service.UserService;
-import com.example.demo.Controller.Models.User;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
 
 @RestController
 @RequestMapping("/users")
+@AllArgsConstructor
 public class ApiController {
-    private final UserService userService;
 
-    @Autowired
-    public ApiController(UserService userService){
```

Comment:
It's better to use `@AllArgsConstructor` to get rid of redundant boiler plate code